crates/forge/src/init.rs

```diff
@@ -60,6 +61,13 @@ fn add_target_to_toml(path: &Path) -> Result<()> {
     Ok(())
 }
 
+fn extend_gitignore(path: &Path) -> Result<()> {
+    let mut file = OpenOptions::new().write(true).append(true).open(path)?;
+    writeln!(file, ".prev_tests_failed")?;
```

Comment:
If we already have `.snfoundry_cache` directory, wouldn't it make sense to store this file there?

Reply:
It's clean after each run of snforge